codesystems/CodeSystem-England-PeriodType.xml

```diff
@@ -0,0 +1,31 @@
+<CodeSystem xmlns="http://hl7.org/fhir">
+	<id value="England-PeriodType"/>
+	<url value="https://fhir.nhs.uk/England/CodeSystem/England-PeriodType"/>
+	<version value="1.0.0"/>
+	<name value="EnglandPeriodType"/>
+	<title value="England Period Type"/>
+	<status value="draft" />
+	<date value="2023-10-13" />
+	<publisher value="NHS England" />
+	<contact>
+		<name value="NHS England" />
+		<telecom>
+			<system value="email" />
+			<value value="interoperabilityteam@nhs.net" />
+			<use value="work" />
+			<rank value="1" />
+		</telecom>
+	</contact>
+	<description value="CodeSystem to identify the type of period data being recorded."/>
+	<copyright value="Copyright &#169; 2021+ HL7 UK Licensed under the Apache License, Version 2.0 (the &quot;License&quot;); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an &quot;AS IS&quot; BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7&#174; FHIR&#174; standard Copyright &#169; 2011+ HL7 The HL7&#174; FHIR&#174; standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html." />
+	<caseSensitive value="true"/>
+	<content value="complete"/>
+	<concept>
+		<code value="legal"/>
+		<display value="Legal"/>
+	</concept>
+	<concept>
+		<code value="operational"/>
+		<display value="Operational"/>
+	</concept>
```

Comment:
Would be worth saying where these two codes come from

Reply:
Agreed, source added.